packages/components/src/core/ButtonIcon/index.stories.tsx

```diff
@@ -195,6 +210,246 @@ export const LivePreview = {
   render: (args: Args) => <LivePreviewDemo {...args} />,
 };
 
+// Screenshot test
+
+const ScreenshotTestDemo = (): JSX.Element => {
+  const topLevel: React.CSSProperties = {
+    columnGap: "20px",
+    display: "inline-grid",
+    fontFamily: "sans-serif",
+    marginRight: "50px",
+  };
+  const displayContents: React.CSSProperties = {
+    display: "contents",
+  };
+  const penultimateLevel: React.CSSProperties = {
+    display: "contents",
+  };
+  const bottomLevel: React.CSSProperties = {
+    marginBottom: 10,
+  };
+  const topLabel: React.CSSProperties = {
+    fontSize: "2em",
```

Comment:
Assigns font-size to the top-level label, since all the labels use `<p>` tags now instead of `<h1>`, etc, so that labels can still be conditional and pass the a11y test (which requires headings to only increase by one level, not possible in the case of conditional heading elements)